homeassistant/components/binary_sensor/rfxtrx.py

```diff
@@ -47,17 +53,16 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
             continue
 
         if entity[ATTR_DATA_BITS] is not None:
-            _LOGGER.info("Masked device id: %s",
-                         rfxtrx.get_pt2262_deviceid(device_id,
-                                                    entity[ATTR_DATA_BITS]))
+            _LOGGER.debug("Masked device id: %s",
+                          rfxtrx.get_pt2262_deviceid(device_id,
+                                                     entity[ATTR_DATA_BITS]))
 
-        _LOGGER.info("Add %s rfxtrx.binary_sensor (class %s)",
-                     entity[ATTR_NAME], entity[CONF_DEVICE_CLASS])
+        _LOGGER.debug("Add %s rfxtrx.binary_sensor (class %s)",
+                      entity[ATTR_NAME], entity[CONF_DEVICE_CLASS])
 
         device = RfxtrxBinarySensor(event, entity[ATTR_NAME],
                                     entity[CONF_DEVICE_CLASS],
                                     entity[ATTR_FIREEVENT],
-                                    entity[ATTR_OFF_DELAY],
```

Comment:
Why are you removing this functionality?

Reply:
It's not allow to add logic into home-assistant. Solve this with platform library